mesheryctl/internal/cli/root/completion.go

```diff
@@ -42,15 +42,15 @@ const example = `  # bash <= 3.2
   # Might need to start a new shell for this setup to take effect.
   $ echo "autoload -U compinit; compinit" >> ~/.zshrc
 
-  source <(mesheryctl system completion zsh)
+  source <(mesheryctl completion zsh)
 
   # zsh on osx / oh-my-zsh
-  mesheryctl system completion zsh > "${fpath[1]}/_mesheryctl"
+  COMPLETION_DIR=$(echo $fpath | grep -o '[^ ]*completions' | grep -v "cache") && mesheryctl completion zsh > "${COMPLETION_DIR}/_mesheryctl"
```

Comment:
there is some issue here. I did not have $HOME/.oh-my-zsh/completions directory on my local m/c. It was failing to define COMPLETION_DIR because of it. Please have a fall back for COMPLETION_DIR

Reply:
Fixed the command to create the dir if it doesn't exist.

Can you try this command instead?
 `COMPLETION_DIR=$(echo $fpath | grep -o '[^ ]*completions' | grep -v cache) && mkdir -p $COMPLETION_DIR && mesheryctl completion zsh > "${COMPLETION_DIR}/_mesheryctl"`